native/src/driver_app.erl

```diff
@@ -0,0 +1,53 @@
+-module(driver_app).
+
+-behaviour(application).
+
+%% Application callbacks
+-export([start/2, stop/1,read_input/0]).
+
+
+
+start(_StartType, _StartArgs) ->
+    read_input(),
+    driver_sup:start_link().
+
+stop(_State) ->
+    ok.
+
+%%====================================================================
+%% Internal functions
+%%====================================================================
+
+%% Read input as a string
+read_input() ->
```

Comment:
This function does not do what it says. It says "Read input as a string", but it seems it also tokenizes and parses.